src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,60 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab: Tab, index: number) => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                key={id}
+                className={classNames(
+                  selectedTabId === id
+                  || (
+                    !tabs.some((searchTab) => searchTab.id === selectedTabId)
+                    && index === 0
+                  )
+                    ? 'is-active' : '',
```

Comment:
it is totally unclear from your code which style will be used here. Think of a way to make it readable
I would come up with creating separate variables for that